schema-registry/common/src/test/resources/avro/nested/nested-complex.avsc

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
Could you move this file to /resources/avro/composites/ ?

Reply:
It seemed like that folder was for testing schemas that leverage Hortonworks "includeSchemas" feature, so I thought keeping schemas related to "nesting" somewhere else would keep things more organized.

Let me know what you think; I can change it if you prefer composites and nested kept together.